homework/test/test-list.js

```diff
@@ -0,0 +1,13 @@
+describe('Test > ', function() {
+  beforeEach(function() {
+	  subject = new ContentManager();
+  });
+
+  it('Test resetWrapper' function() {
+    // Write any pure function assertion here.
+	subject._wrapper = document.createElement('div');
+	subject._wrapper.innerHTML = 'Test';
+	subject.resetWrapper();
+	assert.equal(subject._wrapper.innerHTML, '', "Failed~"
```

Comment:
...typo...

Although you did this part correctly, and with a real method, this defect is really nasty.